config/routes.rb

```diff
@@ -15,6 +15,7 @@
   namespace :admin do
     get '/dashboard', to: 'users#dashboard', as: :dashboard
     resources :merchants, only: [:show, :update]
+    # patch '/merchants/:id/disable', to: 'merchants#update', as: :disable_merchant
```

Comment:
What's this commented out line for?

Reply:
in case we wanted to switch back to a non restful

Reply:
Gotcha